### ISSUE TITLE
Accessibility - HTML validation warnings

### ISSUE THREAD

**Issue by MalcolmShazell** [epimorphics/ukhpi#154]
From the main app page the following warnings generated

Warning: The banner role is unnecessary for element header.

    From line 1, column 3908; to line 1, column 3987
    bsite lr'><header class='with-proposition c-hmlr-header' id='global-header' role='banner'><div c

2.Warning: The main role is unnecessary for element main.

    From line 1, column 5196; to line 1, column 5243
    div></div><main class='container o-container' role='main'><div c

Warning: The contentinfo role is unnecessary for element footer.

    From line 1, column 60585; to line 1, column 60654
    iv></main><footer class='footer group js-footer' id='footer' role='contentinfo'><div c

**Comment by ijdickinson** [comment]
Should be fixed now

**Comment by MalcolmShazell** [comment]
confirm fixed
